pkg/README.md

```diff
@@ -0,0 +1,261 @@
+# cookie
```

Comment:
Is there a method to implement this that does not require maintain two identical copies of the README in the source control? That seems like a lot to do in order to add this `export`/`import` feature.

Reply:
Yes, runing `npm run build` you build all pkg folder. I include it in the repo because I not sure how to build it in old versions of node.